sections/Pipeline.tex

```diff
@@ -5,21 +5,23 @@ \section{Method}\label{sec:method}
 We start with a dataset, in our case consisting of news articles from the media group Nordjyske. 
 Their primary focus is to maintain a variety of local newspapers within the North Jutland region of Denmark. 
 The data ranges from 2017, where a total of $\sim$~63.000 articles have been extracted from their database.
+\vejleder[inline]{ryk det omhandlende datasættet til senere måske experiment}
```

Comment:
skulle mene at vi allerede havde snakket om og blevet enige om at vi referer flere gang til datasættet så vi havde brug for det tidligt?

Reply:
true

Reply:
```suggestion
```
?